app/views/policy_assessments/show_class.html.erb

```diff
@@ -0,0 +1,80 @@
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+
+<% content_for :title, "Assess" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Assess - <%= @klass %></h1>
+
+    <p class="govuk-body">
+      Please indicate if the application complies, does not comply, or
+      is not applicable to each of the policies. Policies are defined in the Town and Country
+      Planning (General Permitted Development) (England) Order 2015,
+      Schedule 2, <%= @klass %>.</p>
+
+    <p class="govuk-body">
+      <%= link_to "Open legislation in new window", @klass.url, class: "govuk-link" %>
```

Comment:
Needs a target, as it currently opens in the same tab

Reply:
Thanks